### ISSUE TITLE
liboqs.pc: incorrect libdir value

### ISSUE THREAD

**Issue by vt-alt** [open-quantum-safe/liboqs#1495]
**Describe the bug**
```
builder@x86_64:/.in$ grep libdir /usr/lib64/pkgconfig/liboqs.pc
libdir=${prefix}/lib
Libs: -L${libdir} -loqs
```
I think (for ALT) proper `libdir` value should be `/usr/lib64` for x86_64. Usually it should have `lib64` for 64-bit systems and `lib` for 32-bit systems. Except for Debian based systems where is different convention — there it will be `lib/x86_64-linux-gnu` perhaps.)

This difference, though, does not produce incorrect flags (on ALT):
```
builder@x86_64:/.in$ pkg-config liboqs --libs
-loqs
```
This is because `-l/usr/lib` is stripped by default by `pkg-config` (based on (default) value of `PKG_CONFIG_SYSTEM_LIBRARY_PATH` containing `/usr/lib`.

So this is **not a major** bug, but better be fixed for correctness in future versions.

BTW. It's possible to force `pkg-config` to not strip system dirs:
```
builder@x86_64:/.in$ PKG_CONFIG_ALLOW_SYSTEM_LIBS=1 pkg-config liboqs --libs
-L/usr/lib -loqs
```
So this could in theory produce erroneous linking for some very non-standrad build environments.

**Environment (please complete the following information):**
 - OS: ALT
 - liboqs version 0.8.0

**Comment by vt-alt** [comment]
Change like this seems to produce correct values:
```diff
-libdir=${prefix}/lib
-includedir=${prefix}/include
+libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
```

**Comment by baentsch** [comment]
> Change like this seems to produce correct values:

Thanks for letting us know. Would you want to contribute the above via PR?

**Comment by vt-alt** [comment]
I will send it later, when I test it on Debian.

**Comment by dstebila** [comment]
Fixed by #1496.
